src/Misc/environmentTextureTools.ts

```diff
@@ -113,7 +135,7 @@ export class EnvironmentTextureTools {
      * @param data The array buffer containing the .env bytes.
      * @returns the environment file info (the json header) if successfully parsed.
      */
-    public static GetEnvInfo(data: ArrayBufferView): Nullable<EnvironmentTextureInfo> {
+    public static GetEnvInfo(data: ArrayBufferView): Nullable<Required<EnvironmentTextureInfo>> {

```

Comment:
Why adding Required?

Reply:
The type for `imageType` is optional [here](https://github.com/BabylonJS/Babylon.js/pull/11102/files#diff-0f4bb579a87a4d0c2f5f84ae1e50a6e14e275e51855ed3b3ad8ac3ab31937a7fR52), as any existing .env file will not have this set, so we should reflect that in the type. But the return value of `GetEnvInfo` is guaranteed to have it, as we are normalizing it by applying the default if not present [here](https://github.com/BabylonJS/Babylon.js/pull/11102/files#diff-0f4bb579a87a4d0c2f5f84ae1e50a6e14e275e51855ed3b3ad8ac3ab31937a7fR164-R166). So using `Required` was the easiest way to turn `imageType?: string` into `imageType: string`. Does that make sense?

Reply:
makes sense thanks!